inst/help/ClassicProcess.md

```diff
@@ -60,7 +60,7 @@ This section allows users to specify multiple process models through one of two
 		- Moderator Z: Second moderator variable.
 
 - Residual Covariances: Adds covariances between the residuals of variables indicating that they have a common cause not specified in the model. Also adds variances for independent (exogenous) variables. This adds degrees of freedom corresponding to the number of covariances and variances added.
-- Parameter Estimates: Determines which parameter estimates are shown in the output.
+- Parameter Estimates: Determines which parameter estimates are shown in the output. Requesting intercepts adds the mean structure to the model.
```

Comment:
When adding the mean structure, the AIC and BIC change because the model has more parameters. All parameter estimates stay the same of course, but it might still confuse people (I was confused at first at least). Perhaps we should add a note on this in the help file?

Reply:
I'll add some more details to this sentence